eternum/src/components/buildings.cairo

```diff
@@ -0,0 +1,24 @@
+//
+// ---------- Buildings
+// NB: Have left this open and not specifcially tied to only Realms. Barbarians could share the same buildings.
+
+#[derive(Component)]
+struct Buildings {
+    barracks: felt252,
```

Comment:
What does the value here represent? How many instances there are?

Reply:
this represents a timestamp, which is then computed to represent how long the building will stand for

Reply:
what would this type expose?

Reply:
it could implement things like `from_unix` `from_datetime` `elapsed` ect.

even without any methods though, it is useful for providing context through the type system

Reply:
Maybe even better would be creating a `TimestampTrait` so we can implement it on other types as well? 

BTW we can alias types in Cairo 1: `type Timestamp = felt252`;